pkg/ccl/backupccl/testdata/backup-restore/show-backup-multiregion

```diff
@@ -39,3 +39,25 @@ SELECT * FROM [SHOW REGIONS FROM DATABASE d] ORDER BY region;
 d eu-central-1 false false {eu-central-1}
 d us-east-1 true false {us-east1}
 d us-west-1 false false {us-west1}
+
```

Comment:
Could you add a case of locality aware backups using the old show backup style as well? (i feel sad to make you do this, given I introduced this bug, but I think it's necessary)

Reply:
argh! nvm. we dont need this, as we never supported old style locality aware show backup